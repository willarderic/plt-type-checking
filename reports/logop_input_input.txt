TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: OR  LEXEME: or						TOKEN: OR        LEXEME: or
LOG_OP -> or							LOG_OP -> or
BIN_OP -> LOG_OP						BIN_OP -> LOG_OP
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: INPUT  LEXEME: input					TOKEN: INPUT     LEXEME: input
INPUT_EXPR -> input						INPUT_EXPR -> input
PARENTHESIZED_EXPR -> INPUT_EXPR				PARENTHESIZED_EXPR -> INPUT_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: INPUT  LEXEME: input					TOKEN: INPUT     LEXEME: input
INPUT_EXPR -> input						INPUT_EXPR -> input
PARENTHESIZED_EXPR -> INPUT_EXPR				PARENTHESIZED_EXPR -> INPUT_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR				PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN  LEXEME: )					TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )					EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR							START -> EXPR

---- Completed parsing ----					---- Completed parsing ----

