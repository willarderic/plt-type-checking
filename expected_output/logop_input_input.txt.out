TOKEN: LPAREN    LEXEME: (
TOKEN: OR        LEXEME: or
LOG_OP -> or
BIN_OP -> LOG_OP
TOKEN: LPAREN    LEXEME: (
TOKEN: INPUT     LEXEME: input
INPUT_EXPR -> input
PARENTHESIZED_EXPR -> INPUT_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
TOKEN: LPAREN    LEXEME: (
TOKEN: INPUT     LEXEME: input
INPUT_EXPR -> input
PARENTHESIZED_EXPR -> INPUT_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
PARENTHESIZED_EXPR -> ARITHLOGIC_EXPR
TOKEN: RPAREN    LEXEME: )
EXPR -> ( PARENTHESIZED_EXPR )
START -> EXPR

---- Completed parsing ----

